ReSharper.FSharp/src/FSharp.Psi.Services/src/Util/ClassRepresentationUtil.fs

```diff
@@ -0,0 +1,15 @@
+module JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util.ClassRepresentationUtil
+
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Parsing
+open JetBrains.ReSharper.Plugins.FSharp.Psi.Tree
+open JetBrains.ReSharper.Psi.Tree
+
+let isEmptyClassRepr (typeRepr: ITypeRepresentation) =
+    match typeRepr with
+    | :? IClassRepresentation as classRepr ->
+        isNotNull classRepr.FirstChild
+        && isNotNull classRepr.LastChild
+        && classRepr.FirstChild.GetTokenType() = FSharpTokenType.CLASS
+        && classRepr.LastChild.GetTokenType() = FSharpTokenType.END
```

Comment:
Are there any cases when class representations have different keywords?

Reply:
It started as a guard agains `struct`, but that's already covered here.